src/formatHit.js

```diff
@@ -31,7 +37,10 @@ export default function formatHit({
     const value = templates.inputValue(suggestion);
     const dropdownValue = templates.dropdownSuggestion({
       ...suggestion,
-      name: hit._highlightResult.locale_names[0].value
+      name: highlightedName,
+      administrative: highlightedAdministrative,
+      city: highlightedCity,
+      country: hit._highlightResult.country.value
```

Comment:
This `hit._highlightResult.country` is `undefined` on countries object, breaking the "Country search" example @vvo -> sorry for not spotting it while reviewing.